liquibase-core/src/main/java/liquibase/database/DatabaseFactory.java

```diff
@@ -291,23 +292,26 @@ private Properties buildDriverProperties(String username, String password, Strin
                 driverProperties.put("password", password);
             }
             if (null != driverPropertiesFile) {
-                File propertiesFile = new File(driverPropertiesFile);
-                if (propertiesFile.exists()) {
-                    LOG.fine(
-                            "Loading properties from the file:'" + driverPropertiesFile + "'"
-                    );
-                    FileInputStream inputStream = new FileInputStream(propertiesFile);
-                    try {
-                        driverProperties.load(inputStream);
-                    } finally {
-                        inputStream.close();
+                try {
+                    PathHandlerFactory pathHandlerFactory = Scope.getCurrentScope().getSingleton(PathHandlerFactory.class);
+                    Resource driverProperty = pathHandlerFactory.getResource(driverPropertiesFile, true);
+                    if (driverProperty != null) {
+                        InputStream stream = driverProperty.openInputStream();
+                        LOG.fine(
+                                "Loading properties from the file:'" + driverPropertiesFile + "'"
+                        );
+                        driverProperties.load(stream);
+                        stream.close();
```

Comment:
Can you do a try with resources here so the stream is always closed?

Reply:
Updated!